@uportal/esco-content-menu/src/components/ContentGrid.vue

```diff
@@ -194,6 +194,10 @@ export default {
       );
     },
   },
+  beforeMount() {
+    window.addEventListener('gridNeedCategories', this.emitAllCategories);
+    window.addEventListener('gridCatFilter', this.setFilterCategory);
```

Comment:
```suggestion
    window.addEventListener('gridCategoryFilter', this.setFilterCategory);
```

Reply:
Seems pedantic given the context, but I'll make the change